styles/productListing.css

```diff
@@ -0,0 +1,67 @@
+.productlisting_banner_container {
+  height: 45vh;
+  background: url(https://cdn.shopify.com/s/files/1/0317/0687/3992/files/bg_about.jpg?v=1580377813);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  margin: 1rem 1.5rem;
+}
+
+.productlisting_main_content_container {
+  margin: 3rem 1.5rem;
+}
+
+.productlisting_sidebar {
+  width: 25%;
+  padding: 1rem 2rem 1rem 1rem;
+  margin-right: 2rem;
+}
+
+.productlisting_products {
+  width: 75%;
+}
+
+.products_filter_section {
+  display: flex;
```

Comment:
Can see that you are having flex Utility Classes Can't we use it here 
Just asking If it is Possible

Reply:
Everything Else Looks Good

Reply:
Did those changes, thanks for the feedback